internal/stage_e7.go

```diff
@@ -0,0 +1,26 @@
+package internal
+
+import (
+	"fmt"
+
+	"github.com/codecrafters-io/interpreter-tester/internal/interpreter_executable"
+	testcases "github.com/codecrafters-io/interpreter-tester/internal/test_cases"
+
+	"github.com/codecrafters-io/tester-utils/test_case_harness"
+)
+
+func testEvaluateConcat(stageHarness *test_case_harness.TestCaseHarness) error {
+	b := interpreter_executable.NewInterpreterExecutable(stageHarness)
+
+	logger := stageHarness.Logger
+
+	concat1 := fmt.Sprintf("\"%s\" + \"%s\"", getRandString(), getRandString())
+	concat2 := fmt.Sprintf("\"%s\" + \"%s\"", getRandString(), getRandIntAsString())
+	concat3 := fmt.Sprintf("\"%s\" + \"%s\" + \"%s\"", getRandString(), getRandString(), getRandString())
+	concat4 := fmt.Sprintf("\"%s\" + \"%s\" + \"%s\" + \"%s\"", getRandString(), getRandString(), getRandString(), getRandString())
```

Comment:
We should we able to test with parentheses at this point, no? (for the 4th combination test)